src/ayesql_parser.yrl

```diff
@@ -1,33 +1,48 @@
 Nonterminals queries named_queries named_query fragments.
-Terminals '$name' '$docs' '$fragment' '$named_param'.
+Terminals '$name' '$type' '$docs' '$fragment' '$named_param'.
```

Comment:
@dkuku I'm sorry, but I don't fully understand the purpose of this `type` in the AyeSQL language. Can you please elaborate? I understand a possible need to parameterize scripts, but I don't think modifying the parser is a good approach.

Reply:
Sorry, this was an accidental commit. It should not be here. But to explain it - some of the libraries accept a modifier for the query:
`-- name: my_query^` means select one similar to Repo.one
`-- name: seed#` allows to run multiple queries from a file - I'm using this for seeding database for tests in one of my projects. I just don't like the modifier and think that having additional comment makes more sense. Especially that function names with `!` have special meaning in elixir.
https://nackjicholson.github.io/aiosql/defining-sql-queries.html
https://github.com/krisajenkins/yesql#insert-returning-autogenerated-keys